src/mpyl/utilities/s3/__init__.py

```diff
@@ -15,17 +15,18 @@
 
 @dataclass
 class S3ClientConfig:
-    service_name = "s3"
-    region_name = "eu-central-1"
-    access_key_id: Optional[str]
-    secret_access_key: Optional[str]
     bucket_name: str
     bucket_root_path: str
+    service_name: str = field(default="s3")
+    region_name: str = field(default="eu-central-1")
+    access_key_id: Optional[str] = field(default=None)
+    secret_access_key: Optional[str] = field(default=None)
```

Comment:
`default=None` seems weird, isn't `None` already the default by default 😁

Reply:
updated

Reply:
oh, they actually don't get initialised to None by default, so it has to be specified. I just tested it again locally after removing the default value